airflow/providers/amazon/aws/operators/eks.pyi

```diff
@@ -0,0 +1,97 @@
+from typing import Optional, Dict, List
```

Comment:
No License info needed?

Reply:
Rebasing to latest main and running pre-commit should add licence automatically.

Reply:
Done, thanks.